keyboards/planck/keymaps/mwpeterson/keymap.c

```diff
@@ -35,6 +35,17 @@ enum planck_layers {
 #define WM_W    LALT(LGUI(KC_LEFT))
 #define WM_CNTR LALT(LGUI(KC_C))
 
+// Unicode
+#ifdef UNICODEMAP_ENABLE
```

Comment:
```suggestion
```
There's no need to put this in an `#ifdef`. If UNICODEMAP isn't enabled, your code will fail to compile anyway because `X()` won't be defined.